src/Microsoft.AspNetCore.SystemWebAdapters.CoreServices/SessionAttribute.cs

```diff
@@ -2,13 +2,51 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Web.SessionState;
 
 namespace Microsoft.AspNetCore.SystemWebAdapters;
 
 [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
 public sealed class SessionAttribute : Attribute
```

Comment:
@Clounea could you add some tests for this? other than that, I'm good with these changes

Reply:
Yes, I could. I will also add tests for overriding scenario